ocaml/idl/datamodel_pool.ml

```diff
@@ -1539,6 +1539,16 @@ let get_guest_secureboot_readiness =
     ~result:(pool_guest_secureboot_readiness, "The readiness of the pool")
     ~allowed_roles:_R_POOL_OP ()
 
+let start_ssh =
+  call ~name:"start_ssh" ~doc:"Start and enable ssh service" ~lifecycle:[]
```

Comment:
I think the following need to be mentioned:
`a helper to call host.start_ssh for all the hosts in the pool`